Steepfile

```diff
@@ -8,6 +8,13 @@ target :lib do
   check "lib/lrama/grammar/code.rb"
   check "lib/lrama/grammar/counter.rb"
   check "lib/lrama/grammar/error_token.rb"
+  check "lib/lrama/grammar/parameterizing_rules/builder.rb"
```

Comment:
It would be better to write `"lib/lrama/grammar/parameterizing_rules"` to detect when files are added to the directory.

Reply:
Sound reasonable.

Reply:
Indeed it is. Thank you! I updated it.